From: Sebastian Andrzej Siewior <sebastian@breakpoint.cc>
Date: Sun, 5 Jul 2020 20:52:39 +0200
Subject: Configurations: PowerPC is big endian

Define B_ENDIAN on PowerPC because it is a big endian architecture. With
this change the BN* related tests pass.

Fixes: #12199

Signed-off-by: Sebastian Andrzej Siewior <sebastian@breakpoint.cc>
---
 Configurations/10-main.conf | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Configurations/10-main.conf b/Configurations/10-main.conf
index 3332f0376508..20dc2db901b8 100644
--- a/Configurations/10-main.conf
+++ b/Configurations/10-main.conf
@@ -706,6 +706,7 @@ my %targets = (
         inherit_from     => [ "linux-generic32" ],
         asm_arch         => 'ppc32',
         perlasm_scheme   => "linux32",
+        lib_cppflags     => add("-DB_ENDIAN"),
     },
     "linux-ppc64" => {
         inherit_from     => [ "linux-generic64" ],
